Api/ApiConnection.cs

```diff
@@ -109,11 +109,16 @@ public bool TryRequest<T>(RestRequest request, out T result)
                     return false;
                 }
 
+                if (!restsharpResponse.IsSuccessful)
+                {
+                    Log.Error($"ApiConnect.TryRequest(): Content: {restsharpResponse.Content}");
+                }
+
                 responseContent = restsharpResponse.Content;
                 result = JsonConvert.DeserializeObject<T>(responseContent);
-                if (!result.Success)
+
+                if (result == null || !result.Success)
```

Comment:
Nice catch! 
I was seeing quite a few null refs on these API calls in syslogs 👍